src/main/java/org/prebid/server/bidder/criteo/model/CriteoRequest.java

```diff
@@ -0,0 +1,27 @@
+package org.prebid.server.bidder.criteo.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.prebid.server.proto.openrtb.ext.request.ExtUserEid;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.List;
+
+@Builder
+@Value
+public class CriteoRequest {
+
+    String id;
+
+    CriteoPublisher publisher;
+
+    CriteoUser user;
+
+    @JsonProperty("gdprconsent")
+    CriteoGdprConsent gdprConsent;
+
+    List<CriteoRequestSlot> slots;
+
+    List<ExtUserEid> eids;
+
```

Comment:
Minor. Remove empty line.
Please check all models for this.

Reply:
Done